schema.go

```diff
@@ -720,6 +720,7 @@ func (s *Schema) genSinglePartitionQuery(t *Table, g *Generator, r *rand.Rand, p
 
 	tableName := t.Name
 	partitionKeys := t.PartitionKeys
+	values := valuesWithToken.Value.Copy()
```

Comment:
This second commit is the same as another existing PR: https://github.com/scylladb/gemini/pull/318
So, why adding it also here?

Reply:
Added whole commit  [9050aa4](https://github.com/scylladb/gemini/pull/310/commits/9050aa45411f091b60ccb52bce1aa19e4812755a) from PR: https://github.com/scylladb/gemini/pull/318. There is no way to test these changes without those changes

Reply:
This PR was based on https://github.com/scylladb/gemini/pull/318 and included [9050aa4], as of now it is rebased and commit went away.